runtime/compiler/optimizer/DataAccessAccelerator.cpp

```diff
@@ -196,6 +196,20 @@ int32_t TR_DataAccessAccelerator::performOnBlock(TR::Block* block, TreeTopContai
    int32_t blockResult = 0;
    bool requestOSRGuardRemoval = false;
 
+   /* Disable DAA if vector packed decimal facilities aren't available.
+    * Disabling the DAA optimization for non-vectorized paths temporarily to unblock
+    * off-heap enablement while we continue to investigate the bug in non-vectorized
+    * pdload evaluator.
+    */
+   if (comp()->target().cpu.isZ() && TR::Compiler->om.isOffHeapAllocationEnabled()
+      && !(comp()->target().cpu.supportsFeature(OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL)
+           && comp()->target().cpu.supportsFeature(OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL_ENHANCEMENT_FACILITY)
+           && comp()->target().cpu.supportsFeature(OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL_ENHANCEMENT_FACILITY_2)
+           && comp()->target().cpu.supportsFeature(OMR_FEATURE_S390_VECTOR_PACKED_DECIMAL_ENHANCEMENT_FACILITY_3)))
+      {
+      return blockResult;
```

Comment:
Also - Can you explain why this change needs to be on performOnBlock and not in the perform earlier in this file to prevent it from even iterating blocks ?

Reply:
Right, yup, we can move it up to `perform()`. I'll update the PR.